models/llm/providers/anthropic_provider.py

```
@@ -0,0 +1,103 @@
+            api_key = EnvVarsContainer.get_env_var("ANTHROPIC_API_KEY", required=True)
+        if not self._initialized:
+            self._api_key = api_key
+            self._initialized = True
+
+    def supports_model(self, model_name: str) -> bool:
+        return model_name in self.supported_models
+
+    def format_structured_output(
+        self,
+        response_model: type[BaseModel],
+        model_config: dict[str, Any],
```

Ruff (0.15.10)

[warning] Unused method argument: `model_config`

(ARG002)